syntax = "proto3"; // 声明了protobuf的版本

package protoMsg; // 声明了代码所在的包（对于C++来说是namespace）

// 创建投票
message LoginRequest
{
    string type = 1;
    string uname = 2;    // 1表示第一个字段
    string upwd = 3;
}
